hack/generate.sh

```diff
@@ -67,12 +81,15 @@ parse_flags() {
 }
 
 overwrite_paths() {
-  local which=$WHICH
-  IFS=' ' read -ra entries <<< "$which"
-  for entry in "${entries[@]}"; do
-    which=${which//$entry/./$entry/...}
+  local options=()
+  IFS=' ' read -ra options <<< "$@"
+  local updated_paths=()
+
+  for option in "${options[@]}"; do
+    updated_paths+=("./$option/...")
   done
-  echo "$which"
+
+  echo "${updated_paths[*]}"
```

Comment:
We were wondering what is the differenced between `echo "${updated_paths[*]}"` and `echo "${updated_paths[@]}"` and why the first is used.

Reply:
IIUC `${updated_paths[*]}` treats the entire array as a single string and returns a single string containing all the elements of the array while `${updated_paths[@]}` treats each element of the array as a separate word and returns them individually. But for our case both should work since the output which is echo'ed contains dilimiters. I will update it for consistency.